### DISCUSSION TITLE
想建议OB12标准中事件的platform改成枚举

### DISCUSSION THREAD

**Discussion by cckk4467** [howmanybots/onebot#193] [想法 / 建议]
`platform | string | OneBot 实现平台名称，格式 [_a-z]+`

主要担心目前各个平台OB实现对事实上相同平台的字段选择不统一，希望约束成枚举值方便应用层区分～
不知道是否合理

**Comment by stdrc** [comment]
我觉得这个自然而然形成约定就好，onebot 如果在标准里约定，那以后新出的聊天平台一样可能会不统一

实际上现在的实现的作者都在一个群里，这个统一起来很容易的
